app/server/route-generator.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+/**
+ * Generates express-jefferson route configs from a methods.js and middlewares.js
+ * config file pair, suitable for placement in the 'routes' config block.
+ *
+ * This function will marry an independent name hash with a middleware hash to create a jefferson-compatible 'routes' config block.
+ * The return value is therefore a map of the form:
+ * {
+ *   <path>: {
+ *      <method>: [<middlewares>]
+ *   }
+ * }
+ *
+ * The intention of this is to allow separation of method naming and HTTP config from middleware definitions for use elsewhere (such as link generation).
+ *
+ * The names should be a map of the form:
+ * {
+ *   <name>: {
```

Comment:
Are you using the latest version of jefferson? We switched the routing from:
name: { method: 'get', path: '/my-uri/', middleware: [] }
to:
'my-uri': { get: [getMiddleware], post: [postMiddleware] }